.github/workflows/main.yml

```diff
@@ -65,7 +78,7 @@ jobs:
 
   push_release:
     if: |
-      github.event_name == 'push' &&
+      github.event_name != 'pull_request' &&
```

Comment:
What events does this include? This is mostly out of curiosity.

Reply:
Push and workflow dispatch will make a new release except pull requests because it will fail